Tests/MapboxMapsTests/Helpers/XCTestCase+ResetStubs.swift

```diff
@@ -0,0 +1,28 @@
+import XCTest
+
+extension XCTestCase {
+    /// Reset all stubs in inner properties
+    ///
+    /// Interates through reflection of current `XCTestCase` and lookup
+    /// for `StubProtocol` properties. Then call `reset()` for each of them.
+    func resetStubs() {
```

Comment:
It looks to me that this method would really shine in the scope of a mock - this way we'd have more granular control over which stubs to reset - e.g. `dependencyProvider.resetStubs()`. Then maybe this method can be renamed to `resetAllStubs`.

Reply:
There are too many Mocks in the repo. Just adding a protocol conformance would take half a day 😅 
Renaming method to `resetAllStubs`